iceoryx_posh/include/iceoryx_posh/internal/roudi/service_registry.hpp

```diff
@@ -38,25 +39,40 @@ class ServiceRegistry
     };
 
     using ReferenceCounter_t = uint64_t;
+
     struct ServiceDescriptionEntry
     {
+        ServiceDescriptionEntry(const capro::ServiceDescription& sd, ReferenceCounter_t count)
```

Comment:
It looks like you are always using `1` as count for the `ServiceDescriptionEntry` wouldn't it make more sense to set it always to one and remove the second argument?
If we need it later we always can add it again - but YAGNI.

Reply:
Not really important, but I do not care either way. The ctor was just needed for an emplace.